data_management/tables.py

```diff
@@ -63,7 +63,7 @@ def external_objects_table_data(request):
         'totalNotFiltered': all_objects.count(),
         'rows': [
             {
-                'doi_or_unique_name': '<a href="data_product/%d">%s</a>' % (obj.data_product.id, obj.doi_or_unique_name),
+                'other_unique_name': '<a href="data_product/%d">%s</a>' % (obj.data_product.id, obj.other_unique_name),
```

Comment:
Not sure what this code does here, but is `identifier` missing again?

Reply:
Just bumping this line... shouldn't there be a matching one for `identifier`?

Reply:
This has been resolved (`identifier` addad)